src/workerd/api/actor.h

```diff
@@ -13,27 +13,101 @@
 #include <capnp/compat/http-over-capnp.h>
 #include <workerd/api/http.h>
 #include <workerd/jsg/jsg.h>
+#include <workerd/jsg/function.h>
 
 namespace workerd {
   template <typename T> class IoOwn;
 }
 
 namespace workerd::api {
 
+enum IsInHouse {
```

Comment:
This needs to be `enum class`, otherwise the constants `YES` and `NO` will be declared in `workerd::api`, which seems like namespace pollution.

That said, it looks like `IsInHouse::NO` is not used in this PR? Why was this enum introduced? It seems like if we want to switch to an enum rather than a boolean for this, we should be modifying the `Fetcher` interface, rather than making this specific to `DurableObject`? That refactoring should be a separate commit or PR... I'd stick with just passing `true` for now.

Reply:
> Why was this enum introduced?

I've seen comments on a few PRs about not passing boolean args to function calls and since I was modifying a few  callsites (when passing compat flags around) I figured I might as well switch it. https://github.com/cloudflare/workerd/compare/8a314fc44ebb642333d14d86d448d1ba345c8e75..f2f0ee599e846b97d2b26ecdcf890e6b7e66318c